modules/150-user-authn/crds/doc-ru-dex-authenticator.yaml

```
@@ -3,74 +3,74 @@ spec:
     - name: v1alpha1
       schema: &schema
         openAPIV3Schema:
-          description: |
-            При появлении объекта `DexAuthenticator` в namespace будут созданы:
-            * Deployment с oauth2-proxy и redis;
-            * Service, ведущий на Deployment с oauth2-proxy;
+          description: |-
+            При появлении объекта `DexAuthenticator` в пространстве имен будут созданы:
+            * Deployment с oAuth2 Proxy и Redis;
+            * Service, ведущий на Deployment с oAuth2 Proxy;
             * Ingress, который принимает запросы по адресу `https://<applicationDomain>/dex-authenticator` и отправляет их в сторону Service;
             * Secret'ы, необходимые для доступа к Dex.
 
-            **Внимание.** При перезапуске пода с oauth2-proxy с помощью refresh token'а будут получены и сохранены в память redis актуальные `access token` и `id token`.
+            **Внимание.** При перезапуске пода с oAuth2 Proxy с помощью `refresh` токена в память Redis будут получены и сохранены актуальные `access token` и `id token`.
           properties:
             spec:
               properties:
                 applicationDomain:
-                  description: |
-                    Внешний адрес вашего приложения, с которого пользовательский запрос будет перенаправлен для авторизации в Dex.
+                  description: |-
+                    Публичный домен для приложения, с которого пользовательский запрос будет перенаправлен для авторизации в Dex.
 
-                    **Не должен** содержать HTTP-схему.
+                    Адрес **не должен** содержать HTTP-схему.
                 sendAuthorizationHeader:
-                  description: |
+                  description: |-
                     Флаг, который отвечает за отправку конечному приложению заголовка `Authorization: Bearer`.
 
-                    Включать, **только** если приложение умеет обрабатывать такой заголовок.
+                    Включать, только если приложение умеет обрабатывать такой заголовок.
```

```suggestion
                    Включать, только если приложение умеет обрабатывать такой заголовок.
```
@mortelumina98 or
Тут был акцент на последствиях. Может предупреждение сделать? **Внимание.** Включайте только, если,..